src/main/webapp/MillenniaDev_script.js

```diff
@@ -0,0 +1,122 @@
+//Returns a list of databases from Cloud Spanner as html dropdown
+function getDatabases(){
+  fetch("/databases").then(response => response.json()).then((list) => {
+    let dropdown = document.getElementById('list-databases');
+    dropdown.options.remove(0);
```

Comment:
What does this line do and why?

Reply:
This clears out all the options of a dropdown so that you can manually input options as you see fit. By removing the option at (0) we're clearing space to add a default option like "Select columns" so that it can be modified to stay hidden or cannot be interacted with.